addons/website_forum/controllers/main.py

```diff
@@ -164,23 +172,40 @@ def tag_read(self, q='', l=25, **post):
         )
         return simplejson.dumps(data)
 
-    @http.route(['/forum/<model("forum.forum"):forum>/tag'], type='http', auth="public", website=True)
-    def tags(self, forum, page=1, **post):
-        tags = request.env['forum.tag'].search([('forum_id', '=', forum.id), ('posts_count', '>', 0)], limit=None, order='posts_count DESC')
+    @http.route(['/forum/<model("forum.forum"):forum>/tag','/forum/<model("forum.forum"):forum>/tag/<tag_char>'], type='http', auth="public", website=True)
+    def tags(self, forum, page=1, tag_char=False, **post):
```

Comment:
Currently in master there is no such filter so it loads all tags and if there are lot's of tags(like we have more than 4800 tags in odoo help) then it consume to much time.
So currentlly it is taking first possible char by default.